build-llvm.py

```diff
@@ -315,6 +322,28 @@ def cleanup(build_folder, incremental):
     build_folder.mkdir(parents=True, exist_ok=True)
 
 
+def get_final_stage(args):
+    """
+    Gets the final stage number, which depends on PGO
+    :param args: The args variable generated by parse_parameters
+    :return: The final stage number
+    """
+    if args.pgo:
```

Comment:
```suggestion
    3 if args.pgo else 2
```

Just a personal preference

Reply:
Nice! Done locally, will push after addressing all comments.